src/main/java/com/example/audit/controller/PagesController.java

```diff
@@ -0,0 +1,24 @@
+package com.example.audit.controller;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class PagesController {
+    @Value("${server.host}")
+    private String host;
+
+    @GetMapping("/list")
+    public String list(final Model model) {
+        model.addAttribute("host", host);
```

Comment:
first of all, you do not need to add application properties to model attributes when opening the template. you can access application properties directly from the template.
 also, why do you need separate controllers for pages and separate controllers for data loading through ajax? you can open the page with data in one call